pyproject.toml

```diff
@@ -0,0 +1,144 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "tianshou"
+version = "0.5.1"
```

Comment:
bump version to 0.6.0?

Reply:
I generally wanted to discuss the release mechanism with you. How do you do it currently?

I added support for bumpversion here, so the versioning can be handled automatically. I have a script and some CI workflows for version bumping and releasing packages in other projects, do you think it makes sense to do that in tianshou as well? You could then release by just calling the script.

Here an example of the [script](https://github.com/appliedAI-Initiative/accsr/blob/develop/build_scripts/release-version.sh) in action and [CI workflow](https://github.com/appliedAI-Initiative/accsr/blob/develop/.github/workflows/release.yaml) in action.